pygac_fdr/__init__.py

```diff
@@ -16,8 +16,4 @@
 # You should have received a copy of the GNU General Public License along with
 # pygac-fdr. If not, see <http://www.gnu.org/licenses/>.
 
-from satpy.utils import get_logger
-import pygac_fdr.version
-
-__version__ = pygac_fdr.version.__version__
-log = get_logger('pygac-fdr')
+__version__ = '0.0.1'
```

Comment:
Did you consider using setup tools scm for getting the version from Git tags as we do in SatPy?

Reply:
No, but I'll check it out! Thanks for the suggestion!